src/systemtask/SystemTask.cpp

```diff
@@ -323,7 +328,11 @@ void SystemTask::Work() {
             // if it's in sleep mode. Avoid bricked device by disabling sleep mode on these versions.
             spiNorFlash.Sleep();
```

Comment:
Sleeping the flash here should *technically* be fine as once a screen has initialised it doesn't tend to load any resources after (so the flash isn't needed). But the 12μA to keep it in standby is probably worth it, the LCD uses 100x this current

Reply:
Yep, we could power the spi flash only when we need it, but yeah, power usage of the chip is very low compares to the display or the MCU so I haven't done that yet :-)